runtime/j9vm/javanextvmi.cpp

```diff
@@ -315,9 +324,8 @@ JVM_VirtualThreadMountEnd(JNIEnv *env, jobject thread, jboolean firstMount)
 	j9object_t threadObj = NULL;
 	BOOLEAN runJ9Hooks = FALSE;
```

Comment:
`vmFuncs` is not used only once; suggest `vm->internalVMFunctions` be used there instead.
`rootVirtualThread` and `runJ9Hooks` are unused; please remove those declaractions.